src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/SymbolTable.cs

```diff
@@ -1354,7 +1354,9 @@ internal void AddPredefinedPropertyToSymbolTable(AggregateSymbol type, Name prop
         private void AddPropertyToSymbolTable(PropertyInfo property, AggregateSymbol aggregate)
         {
             Name name;
-            bool isIndexer = property.GetIndexParameters() != null && property.GetIndexParameters().Length != 0;
+            bool isIndexer = property.GetIndexParameters().Length != 0
```

Comment:
I think Length is ok when we know it is an array. "Any" is often faster with various kind of collections/enumerables, but for array we know the "Length != 0" is the simplest way to check for emptiness.

---
In reply to: [129472003](https://github.com/dotnet/corefx/pull/22537#discussion_r129472003) [](ancestors = 129472003)